modules/tinymce/src/core/main/ts/mode/Readonly.ts

```diff
@@ -116,6 +116,9 @@ const registerReadOnlyContentFilters = (editor: Editor) => {
 
 const isClickEvent = (e: Event): e is MouseEvent => e.type === 'click';
 
+const allowedEvents: ReadonlyArray<string> = [ 'copy' ];
+
+const isReadOnlyAllowedEvent = (e: Event) => allowedEvents.includes(e.type);
```

Comment:
This should be using our Katamari APIs instead, so in this case `Arr.contains`.

Additionally there should be a blank line between functions.

Reply:
done